packages/ckb/src/utils/case-parser.ts

```diff
@@ -1,10 +1,10 @@
 import camelcaseKeys from 'camelcase-keys';
 
-export const toCamelcase = (object: any) => {
+export const toCamelcase = (object: unknown) => {
   try {
     return JSON.parse(
       JSON.stringify(
-        camelcaseKeys(object, {
+        camelcaseKeys(object!, {
           deep: true,
         }),
       ),
```

Comment:
I don’t think it’s a good idea to do `JSON.parse` after `JSON.stringify`. It seems like this is just to convert object keys to camelcase. Is it possible to just use `camelcaseKeys`? I don’t quite understand the reason why use stringify and parse here.

Reply:
There is indeed a performance loss, and a more reasonable approach is to use generics to implement type conversion.

I updated the `toCamelcase` function and related code in https://github.com/ckb-cell/rgbpp-sdk/pull/151/commits/aed41d85101aa2dd4031c561deaef9e24ef56296

cc @ahonn